docs/gallery/concept/autogen/graph_builder_concept.py

```diff
@@ -0,0 +1,295 @@
+"""
+==============================================
+Concepts: The Graph Builder
```

Comment:
more consistent with other sections
```suggestion
Graph Builder
```